docs/Fault-Tolerant-Tachyon-Cluster.md

```diff
@@ -3,12 +3,29 @@ layout: global
 title: Fault Tolerant Tachyon Cluster
 ---
 
+# Fault Tolerance
+
+Fault Tolerance in Tachyon is based upon a multi-master approach where multiple master 
+processes are run.  One of these processes is elected the leader and is used by all workers 
+and clients as the primary point of contact.  The other masters act as standbys using the shared 
+journal to ensure that they maintain the same file system metadata as the leader and can 
+rapidly take over in the event of the leader failing.
+
+If the leader fails a new leader is automatically selected from the available standby masters
+and Tachyon proceeds as usual.  Note that while the switchover to a standby master happens 
+clients may experience brief delays or transient errors.
+
+Note that there is currently no fault tolerance for recovering failed workers.
```

Comment:
There is and there isn't at the same time....

When you loose a worker the block data may be missing in the cluster.  When this happens the client (logic is client side) can elect to read the date from the under fs (depending on the original write type). When it does this it can cache the data back in a new node.